ksql-execution/src/main/java/io/confluent/ksql/execution/codegen/CodeGenRunner.java

```diff
@@ -207,6 +210,28 @@ public Void visitColumnReference(
       return null;
     }
 
+    @SuppressWarnings("deprecation")
+    @Override
+    public Void visitDereferenceExpression(final DereferenceExpression node, final Void context) {
```

Comment:
Here, we simply handle the `DereferenceExpression` as though it were the `FETCH_FIELD_FROM_STRUCT` UDF

Reply:
Perhaps we could move the logic of the FetchFieldFromStruct UDF as a simple static method, and have the codegen call that directly? That way we could get rid of the UDF altogether? Maybe one for a further PR though.